pkg/handler/handler.go

```diff
@@ -335,6 +337,37 @@ func (h handler) apbAddSpec(w http.ResponseWriter, r *http.Request, params map[s
 	writeDefaultResponse(w, http.StatusOK, resp, err)
 }
 
+func (h handler) apbRemoveSpec(w http.ResponseWriter, r *http.Request, params map[string]string) {
+	ansibleBroker, ok := h.broker.(broker.DevBroker)
+	if !ok {
+		h.log.Errorf("unable to use broker - %T as ansible service broker", h.broker)
```

Comment:
@shawn-hurley good enough for me. I like explicit. @jmrodri any objection? We should require this going forwards and consider any code that doesn't conform to be out of date.